build.sbt

```diff
@@ -1,9 +1,28 @@
-name := "circe-yaml-root"
+name := "circe-yaml"
+version in ThisBuild := "0.3.0"
 organization in ThisBuild := "io.github.jeremyrsmith"
 description in ThisBuild := "Library for converting between SnakeYAML's AST and circe's AST"
+licenses in ThisBuild += ("Apache-2.0", url("https://www.apache.org/licenses/LICENSE-2.0.html"))
+
+val Versions = new {
```

Comment:
You can't have an `object` in an `sbt` file, and I generally try to avoid having `.scala` build files. The reflexive calls only happen at build time, and SBT is already doing plenty of them, so I don't really worry about it (actually I'm not sure it's reflexive - there's a distinction between a refinement type and a structural type in that regard)